flexmeasures/conftest.py

```diff
@@ -324,7 +325,12 @@ def create_sources(db) -> dict[str, DataSource]:
     db.session.add(seita_source)
     entsoe_source = DataSource(name="ENTSO-E", type="demo script")
     db.session.add(entsoe_source)
-    return {"Seita": seita_source, "ENTSO-E": entsoe_source}
+    schedule_source = DataSource(name="Schedule", type="demo script")
```

Comment:
Let's name the source `SomeScheduler` instead of `Schedule`, and let's set its type to `"scheduler"`.